### DISCUSSION TITLE
Events Ticketing Application: Show Up 😎👋

### DISCUSSION THREAD

**Discussion by wslyvh** [ethereum-optimism/ecosystem-contributions#168] [Builds]
Sup 😎👋

A solution for the Events Ticketing idea as described at
https://contribute.optimism.io/issue/event-ticket-app

I've been working on a ticketing solution called Show Up Protocol
https://github.com/wslyvh/show-up

[Show Up](https://www.showup.events/) is an onchain RSVP and Event management protocol designed to reshape event participation. It introduces a novel staking mechanism that aligns the incentives between event organizers and attendees.

- Higher event participation for event hosts
- Reward active participation for attendees
- A decentralized, open and permissionless protocol

win/win for everyone 🤝

Show Up Protocol is a set of decentralized, open, and permissionless smart contracts built on the Ethereum protocol. These contracts manage commitments and the conditions for keeping them. It is exclusively deployed on [Optimism](https://optimistic.etherscan.io/address/0x27d81f79D12327370cdB18DdEa03080621AEAadC) and [Base](https://basescan.org/address/0x27d81f79D12327370cdB18DdEa03080621AEAadC). The event metadata is stored on IPFS, ensuring the platform has no control over your events, funds, or payments.

*It doesn't use EAS attestations (yet) - altho we have been in touch

**Comment by JSeiferth** [comment]
Heeey @wslyvh! Great to see you here :) 

There's a team working on a Foundation mission that's very similiar, idk if there's an overlap 
https://github.com/ethereum-optimism/ecosystem-contributions/issues/115

**Reply by wslyvh** [reply]
Hi @JSeiferth thank you! Seems like there could indeed be some overlap. I can imagine more that Show Up attendees could classify for larger faucet drips and/or get a similar proof of attendance attestation. Would be interesting to explore an integration. I see a recent updates was posted, but what's the best way to get in touch and follow progress?

**Comment by ribeirojose** [comment]
hey @wslyvh ! I'm working on a POC and was about to claim the idea when I saw this discussion. I noticed you haven't created a PR for this discussion -- is this still something you're interested in pushing forward/maintaining? Would love to join efforts if that makes sense to you!

**Reply by wslyvh** [reply]
Done! Didn't realize it should be updated as PRs

https://github.com/ethereum-optimism/ecosystem-contributions/pull/231

I'd be open to contributions or collaborate if you're interested to add EAS Attestations to it tho!
